test cases/common/205 subproject with features/meson.build

```diff
@@ -0,0 +1,16 @@
+project('proj', 'c')
+
+auto_subproj = subproject('sub', required: get_option('use-subproject'))
+assert(auto_subproj.found(), 'Subproject should always be buildable and thus found')
+
+auto_dep = dependency('', fallback: ['sub', 'libSub'], required: true)
+assert(auto_dep.found() == true, 'Subproject is required and foundable, dependency should be found.')
+
+disabled_subproj = subproject('disabled_sub', required: get_option('disabled-subproject'))
+assert(disabled_subproj.found() == false, 'Disabled subproject should be NOT found')
+
+disabled_dep = dependency('', fallback: ['disabled_sub', 'libSub'], required: false)
```

Comment:
The test should use this dependency in a target to test that you can use not-found not-required fallback dependencies just like not-found-not-required ordinary dependencies.

Reply:
Done.

Reply:
This is causing the test to fail because `nothing.c` doesn't exist. The purpose of this test is to ensure that the target continues to build if an optional dependency is not found.